mail-client/alot/alot-0.9.1.ebuild

```diff
@@ -18,17 +19,16 @@ IUSE="doc test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
-	>=app-crypt/gpgme-1.9.0[python,${PYTHON_USEDEP}]
-	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	app-crypt/gpgme[python,${PYTHON_USEDEP}]
+	dev-python/configobj[${PYTHON_USEDEP}]
 	dev-python/python-magic[${PYTHON_USEDEP}]
-	>=dev-python/urwid-1.3.0[${PYTHON_USEDEP}]
-	>=dev-python/urwidtrees-1.0[${PYTHON_USEDEP}]
+	dev-python/urwid[${PYTHON_USEDEP}]
+	dev-python/urwidtrees[${PYTHON_USEDEP}]
 	>=dev-python/twisted-18.4[${PYTHON_USEDEP}]
 	net-mail/mailbase
-	>=net-mail/notmuch-0.27[crypt,python]
+	net-mail/notmuch[crypt,python]
 	"
 DEPEND="
```

Comment:
This should be BDEPEND. It can be called after distutils_enable_tests, which automatically appends test? ( ${RDEPEND} ) to it.

Reply:
Ok.